benchmark/Streamly/Benchmark/Data/Stream/Eliminate.hs

```diff
@@ -238,6 +257,7 @@ benchPureSink :: NFData b
     => Int -> String -> (SerialT Identity Int -> b) -> Benchmark
 benchPureSink value name = benchPure name (sourceUnfoldr value)
 
+#ifdef USE_PRELUDE
```

Comment:
Can we replace the `t` type in thee routines with `Stream` instead? If so we do not need to use the `USE_PRELUDE` conditional here.